lib/menu.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'send_sms'
+
+class Menu
+
+attr_reader :menu, :selection_of_dishes, :message
+    
+    def initialize
+        @menu = {
+            "lasagne" => 12,
+            "oysters" => 14,
+            "fries" => 4,
+            "green_salad" => 9,
+            "rump_steak" => 20,
+            "tiramisu" => 25
+        }
+        @selection_of_dishes = []
+        #@texter = SendText.new
+    end
+
+    def show_menu
+        puts @menu
```

Comment:
You could get rid of the attr_reader for menu since you have this method that allows the user to see it